fili-core/src/test/groovy/com/yahoo/bard/webservice/web/JobsApiRequestSpec.groovy

```diff
@@ -121,63 +117,6 @@ class JobsApiRequestSpec extends Specification {
         errorSubscriber.getOnErrorEvents().get(0).getMessage() == "No job found with job ticket IDontExist"
     }
 
-    def "handleBroadcastChannelNotification returns an Observable that replays notification if it was sent before async timeout"() {
-        setup:
-        TestSubscriber<PreResponse> testSubscriber = new TestSubscriber<>()
-
-        when:
-        Observable<PreResponse> preResponseObservable = defaultJobsApiRequest.handleBroadcastChannelNotification("ticket1")
-
-        broadcastChannel.publish("ticket1")
-        preResponseObservable.subscribe(testSubscriber)
-
-        then:
-        testSubscriber.assertReceivedOnNext([ticket1PreResponse])
-    }
-
-    def "handleBroadcastChannelNotification returns an Observable that filters notifications and only returns a single notification for the given ticket"() {
-        setup:
-        TestSubscriber<PreResponse> testSubscriber = new TestSubscriber<>()
-
-        when:
-        Observable<PreResponse> preResponseObservable = defaultJobsApiRequest.handleBroadcastChannelNotification("ticket1")
-        broadcastChannel.publish("ticket_before_connecting_is_dropped")
-        broadcastChannel.publish("ticket1")
-        broadcastChannel.publish("ticket2")
-        broadcastChannel.publish("ticket1")
-        preResponseObservable.subscribe(testSubscriber)
-
-        then:
-        testSubscriber.assertReceivedOnNext([ticket1PreResponse])
-    }
```

Comment:
This is subsumed by a [new test in the JobsServletReactiveChainforResultsEndpointSpec](https://github.com/yahoo/fili/pull/39/files#diff-fd9bfd667dcfa0b4c6d56f006840de39R208)